src/components/CommentList.jsx

```diff
@@ -0,0 +1,47 @@
+import { useState, useEffect } from "react";
+import CommentCard from "./CommentCard";
+import { getArticleComments } from "../api";
+import { LoadingAnimation } from "./LoadingAnimation";
+
+export default function CommentList({ article_id }) {
+  const [isLoading, setIsLoading] = useState(false);
+  const [isError, setIsError] = useState(false);
+  const [comments, setComments] = useState([]);
+
+  useEffect(() => {
+    //prevents unresolved article_id being passed to Axios fetcher
+    if (!article_id) return;
+    setIsLoading(true);
+    getArticleComments(article_id)
+      .then(({ comments }) => {
+        setComments(comments);
+        setIsError(false);
+        setIsLoading(false);
+        return comments;
+      })
+      .catch((error) => {
```

Comment:
This catch will never be invoked at the moment due to you having a catch in the api file.